src/Akka.Persistence.Redis/Query/Stages/AllEventsSource.cs

```diff
@@ -74,15 +75,7 @@ private class AllEventsLogic : GraphStageLogic
             private long _toSequenceNr;
             private readonly bool _live;
 
-            public AllEventsLogic(
-                ConnectionMultiplexer redis,
-                int database,
-                Config config,
-                ActorSystem system,
-                long fromSequenceNr,
-                long toSequenceNr,
-                bool live,
-                Outlet<EventEnvelope> outlet, Shape shape) : base(shape)
+            public AllEventsLogic(ConnectionMultiplexer redis, int database, Config config, ActorSystem system, long fromSequenceNr, long toSequenceNr, bool live, Outlet<EventEnvelope> outlet, Shape shape) : base(shape)
```

Comment:
Please don't do this, it makes the code unreadable.

Reply:
Sorry, I had to do that to have a clearer picture. I will revert.